utils/metadata/metadata_function.py

```diff
@@ -0,0 +1,33 @@
+import csv
+from typing import List, Union
+from utils.objects.scene_object import SceneObject
+
+
+
+
+def get_metadata_from_scene_file(path_to_scene_csv: str) -> List[SceneObject]:
+    scene_objects = []
+    with open(path_to_scene_csv, "r") as f:
+        reader = csv.DictReader(f)
+        for scene in reader:
+           scene_objects.append(SceneObject(
+            duration=scene['Length (seconds)'],
+            scene_start=scene['Start Time (seconds)'],
+            scene_end=scene['End Time (seconds)'],
+            title=scene['file_name'],
+        ))
+    return scene_objects
+
+def get_metadata_from_keyframe_file(path_to_keyframes_csv: str, scene_objects: List[SceneObject]) -> List[SceneObject]:
+    with open(path_to_keyframes_csv, "r") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            for scene_object in scene_objects:
```

Comment:
It would be nice if we could make this extendable from the start i.e. give the task dict as a parameter here and search for the keys defined for the tasks dict. That way we dont have to change the code here everytime we make adjustments to the tasks.